### ISSUE TITLE
Citing fancyimpute

### ISSUE THREAD

**Issue by brettbj** [hammerlab/fancyimpute#7]
I'm using fancyimpute in a conference submission. Is there a citation format you'd like followed? I didn't see a paper on first check

**Comment by sergeyf** [comment]
Hey, cool. We should probably do something like Zenodo. @iskandr since you're the owner, would you do the honors? 

BTW, which algorithm did you end up using? Did it work well?

**Comment by brettbj** [comment]
I'm doing a comparison for EHR data (initially clinical trial based) so I used a few (SVD, SI, KNN) . But I had really interesting results with a DA to model EHR data in general (simulated first - http://biorxiv.org/content/early/2016/02/18/039800) 

In your autoencoder is there any reason you used MSE instead of binary cross entropy (are you generalizing to un-normalized data)?

**Comment by sergeyf** [comment]
Yeah, MSE should work with pretty much any data (except categorical). Also, our auto-encoder is not denoising if I recall correctly - would be a good future feature (we also welcome PRs).

**Comment by brettbj** [comment]
Will do. I forked it and will make a PR when it's production ready. For normalized data I do find much better results with bce than mse.

**Comment by sergeyf** [comment]
Awesome, thanks!

Is it sufficient to do something like (x - min(x) ) / (max(x) - min(x)) to use cross-entropy? If so, we could make it an option to specify the loss and then apply that transform to the output data if cross-entropy is chosen.

**Comment by brettbj** [comment]
There are keras functions for both - http://keras.io/objectives/. Maybe that's the best route?

**Comment by iskandr** [comment]
Hey @brettbj, I just added a DOI. Is that enough?

**Comment by brettbj** [comment]
Yes, thanks!
